src/Gui/SoFCColorBar.cpp

```diff
@@ -78,6 +80,32 @@ void SoFCColorBarBase::GLRenderBelowPath(SoGLRenderAction *  action)
     SoSeparator::GLRenderBelowPath(action);
 }
 
+float SoFCColorBarBase::getBoundingWidth(const SbVec2s& size)
+{
+    // These are the same camera settings for front nodes as defined in the 3d view
+    SoOrthographicCamera* cam = new SoOrthographicCamera;
+    cam->position = SbVec3f(0, 0, 5);
+    cam->height = 10;
```

Comment:
Same here, why do we need height 10 or would do any height > 0 do the job?

Reply:
See above. For `height` no arbitrary value should be used because the hard-coded values for the color bar must be relative to the height value. Since in the viewer it's set to 10 we must use 10 here, too.